src/AzureEventGridSimulator/appsettings.json

```diff
@@ -7,7 +7,27 @@
       "subscribers": [
         {
           "name": "LocalAzureFunctionSubscription",
-          "endpoint": "http://localhost:7071/runtime/webhooks/EventGrid?functionName=PersistEventToDb"
+          "endpoint": "http://localhost:7071/runtime/webhooks/EventGrid?functionName=PersistEventToDb",
+          "filter": {
```

Comment:
I think for the F5 experience, I don't think we should have any pre-configured filters in here. Can we remove `filter` from here and add some examples in the `ReadMe` file.